.travis.yml

```diff
@@ -22,6 +30,9 @@ env:
     - COVERALLS_REPO_TOKEN=hnXUEBKsORKHc8xIENGs9JjktlTb2HKlG
     - BOTO_CONFIG=/tmp/nowhere
 language: python
+# We do all builds for both Python 2.7 and 3.4, with the exception of
+# static analysis, which is just run once (and checks with both Python
+# versions).
```

Comment:
"all suites", perhaps? The three things we're doing for each Python version are called "suites" here. On the Travis side, I think an actual running of the tests (a specific suite on a specific version) is a "job", and the thing that consists of one job per entry in this matrix is a "build".

Also I'd s/checks with/checks against/. "Checks with" sounds to me like it's using that version to do the checking.